ntbs-service/Models/Notification.cs

```diff
@@ -227,5 +233,14 @@ private string CreateTimePeriodsString(RiskFactorDetails riskFactor)
             }
             return string.Join(", ", timeStrings);
         }
+
+        private int GetAgeAtTimeOfNotification()
+        {
+            if (NotificationDate == null || PatientDetails?.Dob == null) 
+            {
+                return 0;
+            }
+            return (int) Math.Floor(((DateTime) NotificationDate - (DateTime) PatientDetails.Dob).Days / 365.2425);
```

Comment:
Could we split this into two lines? find it hard to ready because of all the brackets!